tests/unit/components/generic/ContentCard.spec.js

```diff
@@ -9,11 +9,36 @@ const factory = () => shallowMount(ContentCard, {
   localVue
 });
 
-describe('ContentCard', () => {
-  it('includes a "Read more" link', () => {
+describe('components/generic/ContentCard', () => {
+  it('includes a description', () => {
     const wrapper = factory();
+    wrapper.setProps({ description: 'The Milkmaid by Vermeer' });
 
-    const link = wrapper.find('.card-link');
-    link.text().should.include('Read more');
+    const description =  wrapper.find('[data-qa="content card"] .card-text');
+    description.text().should.eq('The Milkmaid by Vermeer');
+  });
+
+  it('has a link', () => {
+    const wrapper = factory();
+    wrapper.setProps({ url: 'https://example.org' });
+
+    const link =  wrapper.find('[data-qa="content card"] .card-link');
+    link.attributes().href.should.eq('https://example.org');
+  });
+
+  it('has an image', () => {
+    const wrapper = factory();
+    wrapper.setProps({ imageUrl: 'https://example.org' });
+
+    const card =  wrapper.find('[data-qa="content card"]');
+    card.attributes().imgsrc.should.eq('https://example.org');
```

Comment:
I would be tempted to do a full mount, not a shallow mount, and test that there is an actual `img` displayed.